<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPlant</class>
 <widget class="QDialog" name="addPlant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(251, 255, 237);</string>
  </property>
  <widget class="QFrame" name="frameAddPlant">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QGraphicsView" name="addPlantImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 6px;
border: 2 solid rgb(255, 170, 127);
background: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(125, 92, 73);
font-size: 16px;
font-weight: bold;
border: none;
</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditNameAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
border: 2 solid #A5D6A7 ;
border-radius: 6px;
color: rgb(170, 170, 127);
font-size: 14px;
font: 9pt &quot;Century Gothic&quot;;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSortAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(125, 92, 73);
font-size: 16px;
font-weight: bold;
border: none;
</string>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSortAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
border: 2 solid #A5D6A7 ;
border-radius: 6px;
color: rgb(170, 170, 127);
font-size: 14px;
font: 9pt &quot;Century Gothic&quot;;
</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinWateringAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
border: 2 solid #A5D6A7 ;
border-radius: 6px;
color: rgb(170, 170, 127);
font-size: 14px;
font: 9pt &quot;Century Gothic&quot;;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWateringAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(125, 92, 73);
font-size: 16px;
font-weight: bold;
border: none;
</string>
    </property>
    <property name="text">
     <string>Watering once/day</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFertilizerAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(125, 92, 73);
font-size: 16px;
font-weight: bold;
border: none;
</string>
    </property>
    <property name="text">
     <string>Fertilizer once/week</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinFertilizerAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
border: 2 solid #A5D6A7 ;
border-radius: 6px;
color: rgb(170, 170, 127);
font-size: 14px;
font: 9pt &quot;Century Gothic&quot;;
</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
border: 2 solid #A5D6A7 ;
border-radius: 6px;
color: rgb(170, 170, 127);
font-size: 14px;
font: 9pt &quot;Century Gothic&quot;;

</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTransplantAddPlant">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(125, 92, 73);
font-size: 16px;
font-weight: bold;
border: none;
</string>
    </property>
    <property name="text">
     <string>Last transplant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddPlantImage">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 194, 217);
border-radius: 10px;
color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="horizontalGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnBackDelete">
      <property name="styleSheet">
       <string notr="true">background-color: #A5D6A7;
color: #2E7D32;
border: none;
border-radius: 6px;
padding: 10px 20px;
font-size: 16px;
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnYesDelete">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 88, 91);
color: white;
border: none;
border-radius: 6px;
padding: 10px 20px;
font-size: 16px;
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
